experiment/build/builder.py

```diff
@@ -22,30 +22,27 @@
 import subprocess
 import sys
 import tarfile
-import tempfile
 import time
-from typing import Callable, Dict, List, Tuple
+from typing import Callable, List, Tuple
 
 from common import benchmark_utils
 from common import experiment_path as exp_path
-from common import experiment_utils
 from common import filesystem
 from common import fuzzer_utils
 from common import gsutil
 from common import logs
-from common import new_process
 from common import utils
+from experiment.build import build_utils
+
+if not utils.is_local_experiment():
+    import experiment.build.gcb_build as buildlib
```

Comment:
Better to do this via classes and inherit gcb and local via some base Build class.

Reply:
those variables weren't needed so essentially functions in *_build won't share state so a class isn't needed. I'm not sure if it would be a cleaner interface, but this seems fine to me (similar to posixpath vs ntpath in os.path).